src/engine/strat_engine/dmdevice.rs

```diff
@@ -0,0 +1,142 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Functions for dealing with device mapper devices.
```

Comment:
It's a whole module...I left open the possibility that Stratis might need to do something else with DM devices besides construct their names.